pkg/apparmor/apparmor_linux_template.go

```diff
@@ -21,6 +21,10 @@ profile {{.Name}} flags=(attach_disconnected,mediate_deleted) {
   # Allow signals from privileged profiles and from within the same profile
   signal (receive) peer=unconfined,
   signal (send,receive) peer={{.Name}},
+  # Allow certain signals from OCI runtimes (podman, runc and crun)
+  signal (receive) peer={/usr/bin/,/usr/sbin/,}runc,
+  signal (receive) peer={/usr/bin/,/usr/sbin/,}crun,
```

Comment:
It's an "AppArmor Regular Expression" which is a glob with knobs on

[Globbing (AARE)](https://man.archlinux.org/man/extra/apparmor/apparmor.d.5.en#Globbing_(AARE))

```
File resources and other parameters accepting an AARE may be specified with a globbing syntax similar to that used by popular shells, such as [csh(1)](https://man.archlinux.org/man/csh.1.en), [bash(1)](https://man.archlinux.org/man/bash.1.en), [zsh(1)](https://man.archlinux.org/man/zsh.1.en).

*
can substitute for any number of characters, excepting '/'
**
can substitute for any number of characters, including '/'
?
can substitute for any single character excepting '/'
[abc]
will substitute for the single character a, b, or c
[a-c]
will substitute for the single character a, b, or c
[^a-c]
will substitute for any single character not matching a, b or c
{ab,cd}
will expand to one rule to match ab, one rule to match cd
Can also include variables.
@{variable}
will expand to all values assigned to the given variable.
When AppArmor looks up a directory the pathname being looked up will end with a slash (e.g., /var/tmp/); otherwise it will not end with a slash. Only rules that match a trailing slash will match directories. Some examples, none matching the /tmp/ directory itself, are:

/tmp/*
Files directly in /tmp.
/tmp/*/
Directories directly in /tmp.
/tmp/**
Files and directories anywhere underneath /tmp.
/tmp/**/
Directories anywhere underneath /tmp.
```

Reply:
Ok change the crun to allow for crun-wasm, crun-qm and other future OCI Runtimes based off of crun.

Reply:
That's done.